build.gradle

```
@@ -106,15 +106,15 @@ buildScan {
 
 ext {
     daggerVersion = '2.22.1'
-    fluxCVersion = '1.5.4'
+    fluxCVersion = '1.5.7'
```

```suggestion
    fluxCVersion = '1.5.8'
```

@maxme Can we bump the FluxC tag to `1.5.8` in this PR? I was going to open a PR for that, but I don't think it makes sense to have 2 PRs bumping FluxC at the same time.

The main difference between `1.5.7` and `1.5.8` should be this PR: https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1468 which should have no effect on WPAndroid. I've tried updating the tag in this branch and didn't see any issues.

---

Sure - done in 44e3425